cartography/intel/aws/route53.py

```diff
@@ -94,8 +95,8 @@ def load_cname_records(neo4j_session, records, update_tag):
 
 def load_zone(neo4j_session, zone, current_aws_id, update_tag):
     ingest_z = """
-    MERGE (zone:DNSZone:AWSDNSZone{name: {ZoneName}})
-    ON CREATE SET zone.firstseen = timestamp(), zone.zoneid = {ZoneId}
+    MERGE (zone:DNSZone:AWSDNSZone{zoneid:{ZoneId}})
```

Comment:
Fix for issue #208

Reply:
The change makes sense to me, but I would get a lot more confidence in this significant schema change (changing a primary identifier that is used in merging) if we added an integration test.

This would involve including a tiny redacted sample of AWS DNS data and showing that it gets loaded to the graph as expected and connected with other nodes as expected.

I'm not saying create tests for everything in this module - just ensure that the nodes that you're changing are still connected to the other nodes we expect them to be connected to.

Reply:
I will look into it because I generally agree on adding integration tests. In this case you can see that every other statement uses the zoneid field which is causing missing relationships. I see this more as a bug fix than a significant schema change.